src/tests/comments.test.js

```diff
@@ -6,83 +6,84 @@ import users from './mock-data/test-users';
 import tokenizer from '../utils/jwt';
 import { hotelfactory, roomfactory } from './scripts/factories';
 import db from '../models';
+import truncate from './scripts/truncate';
 
 use(chaiHttp);
 
-const prefix = '/api/v1';
-let rightUserToken, wrongUserToken, requestId, commentExample;
+describe('', () => {
+  const prefix = '/api/v1';
+  let rightUserToken, wrongUserToken, requestId, commentExample;
 
```

Comment:
Nice work, but I think the `describe` name is missing

Reply:
> Nice work, but I think the `describe` name is missing

Ah ok, let me add a description for the file.

Reply:
Now fixed, thanks @b0nbon1